source/server/hot_restarting_base.cc

```diff
@@ -11,6 +11,8 @@ namespace Server {
 using HotRestartMessage = envoy::HotRestartMessage;
 
 static constexpr uint64_t MaxSendmsgSize = 4096;
+static constexpr std::chrono::milliseconds CONNECTION_REFUSED_RETRY_DELAY{5000};
```

Comment:
does this need to be 5s? I might have tried 1s to see if that helps.Also, if you are using absl::SleepFor, consider declaring this as absl::Duration.

Reply:
I wanted to try something high first to ensure it was good enough, but 1s intervals with a total wait of 10s is probably ok.

I'll move to absl::Duration.